lib/matplotlib/tests/test_style.py

```diff
@@ -160,3 +163,94 @@ def test_xkcd_cm():
     with plt.xkcd():
         assert mpl.rcParams["path.sketch"] == (1, 100, 2)
     assert mpl.rcParams["path.sketch"] is None
+
+
+@check_figures_equal(extensions=["png"])
+def test_seaborn_style(fig_test, fig_ref):
+    seaborn = pytest.importorskip('seaborn')
```

Comment:
I didn't get `check_figures_equal` to work with the `seaborn` fixture, so I copied the checking code here explicitly.